using-mobile-extensions/mobile-core/platform-services.md

```diff
@@ -146,3 +148,128 @@ This step should occur prior to any other interactions with the AEP SDK. While i
     }
 ```
 
+## iOS
+
+### 1. Conform to the ACPHttpConnectionPerformer protocol
+
+The `ACPHttpConnectionPerformer` is a protocol which must be conformed to in order to override the network stack. It provides two methods which must be implemented:
+
+    1. `shouldOverride`, which takes a URL and an HTTP method. You return true if you want the network stack to be overriden     by the performer, or false if not. 
```

Comment:
Take out the "You". Should be "overridden" (with two ds). Also, looks like there is some whitespace between "overridden" and "by"